namespaces/Representor.md

```diff
@@ -0,0 +1,141 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Object Element)
+
+The Resource represents a Hypermedia Resource, with it's available relations, transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `content`: (object)
+    - `relations` - (array[Relation]) - The relations from this resource.
+    - `transitions` - (array[Transition]) - The available transitions from this resource.
+    - `resources` - (array[Resource]) - The embedded relations (resources) from this resource.
+    - `attributes` - (object) - Any properties of the resource.
+    - `relation` - (string) - An optional relation to this resource.
```

Comment:
I am completely missing this one – what is it good for? I assume it says what "relation" was exercised to get to this resource. Is this used anywhere in the representor pattern?